canonical-web-and-design/ubuntu.com
# Create the ubuntu masters virtual event takeover

## Done

- Create the ubuntu masters virtual event takeover
- Added it to the homepage
- Added it to the /takeover page
- **NOTE needs link to brighttalk**

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/ or the bottom of http://0.0.0.0:8001/takeovers
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1Rj_FAPxxyx0SPP5qMOZJpgPCMhK81SBTHz5ZsY6n2z0/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2531

## Screenshots

![image](https://user-images.githubusercontent.com/441217/77669866-54a64780-6f7d-11ea-9f5b-0224d8e85985.png)
